Assets/Scripts/StaminaBarSetColour.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StaminaBarSetColour : MonoBehaviour {
+    public SpriteRenderer spriteRenderer;
```

Comment:
you could've achieved the same outcome much easier with a simple line renderer, then again, this is just a prototype